src/utils.jsx

```diff
@@ -12,4 +12,12 @@ const patchData = (url, body) => {
   });
 };
 
-export { fetchData, patchData };
+const postData = (url, body) => {
+  return fetch(basicURL + url, {
+    method: "POST",
+    headers: { "Content-type": "application/json" },
+    body: JSON.stringify(body),
```

Comment:
It's worth noting that if you switch to `axios`, a lot of these steps can be skipped, but if you're happy with fetch, no problem!